src/api-controller.js

```diff
@@ -18,15 +18,35 @@ function _getDefaultHeaders() {
 }
 
 exports.searchIssues = async function (jql) {
-  const res = await fetch(
-    defaultURL + `/rest/api/3/search?maxResults=${process.env.JIRA_MAX_SEARCH_RESULTS}&jql=${jql}`,
-    {
-      method: 'GET',
-      headers: _getDefaultHeaders(),
-      agent: httpsAgent,
+  let formattedRes;
+  let startAt = 0;
+  let pageCount = 1;
+  let issueList = [];
+
+  while (true) {
+    if (pageCount === 0) break;
+
+    let response  = await fetch(
+      defaultURL + `/rest/api/3/search?maxResults=${process.env.JIRA_MAX_SEARCH_RESULTS}&startAt=${startAt}&jql=${jql}`,
```

Comment:
since there is a max value for ``JIRA_MAX_SEARCH_RESULTS`` please include a comment in the env-sample file indicating the max value. or else we can remove that variable and hardcode the value in js file